.circleci/config.yml

```diff
@@ -0,0 +1,78 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: mozilla/cidockerbases:docker-latest
+    working_directory: /
+
+    steps:
+      - run:
+          name: Get info
+          command: |
+              uname -v
+              docker info
+
+      - run:
+          name: Install essential packages
+          command: apt-get install make
+
+      - checkout:
+          path: /oidc_testprovider
+
+      - run:
+          name: Build Docker images
+          working_directory: /oidc_testprovider
+          command: |
+            make build
+
```

Comment:
At some point, we might want to run tests or linting or something here. We can figure that out in a future PR.